packages/query/src/index.ts

```diff
@@ -849,7 +878,7 @@ export const ${camel(
     `${operationPrefix}-${type}`,
   )}(${queryOptionsVarName}) as ${returnType};
 
-  query.queryKey = ${queryOptionsVarName}.queryKey;
+  query.queryKey = ${queryOptionsVarName}.queryKey as QueryKey;
```

Comment:
This seems to apply to `samples/react-query/hook-mutator/endpoints.ts` as well as vue one, not sure if that was your intention.